registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/abstractverticle/MosipVerticleAPIManager.java

```diff
@@ -1,227 +1,243 @@
+					servletPath.substring(servletPath.lastIndexOf("/") + 1, servletPath.length()) + "Verticle",
+					healthCheckTimeOut,
+					future -> healthCheckHandler.senderHealthHandler(future, vertx, super.mosipEventBus, sendAddress));
+		}
+		if (servletPath.contains("uploader")) {
+			healthCheckHandler.register("virusscanner", healthCheckTimeOut, healthCheckHandler::virusScanHealthChecker);
+			healthCheckHandler.register(
+					servletPath.substring(servletPath.lastIndexOf("/") + 1, servletPath.length()) + "Consume",
+					healthCheckTimeOut, future -> {
+						healthCheckHandler.consumerHealthHandler(future, vertx, super.mosipEventBus, consumeAddress);
+					});
+			healthCheckHandler.register(
+					servletPath.substring(servletPath.lastIndexOf("/") + 1, servletPath.length()) + "Verticle",
+					healthCheckTimeOut,
+					future -> healthCheckHandler.senderHealthHandler(future, vertx, super.mosipEventBus, sendAddress));
+		}
+		if (checkServletPathContainsCoreProcessor(servletPath)) {
+			healthCheckHandler.register(
+					servletPath.substring(servletPath.lastIndexOf("/") + 1, servletPath.length()) + "Send",
+					healthCheckTimeOut, future -> {
+						healthCheckHandler.senderHealthHandler(future, vertx, super.mosipEventBus, sendAddress);
+					});
+			healthCheckHandler.register(
+					servletPath.substring(servletPath.lastIndexOf("/") + 1, servletPath.length()) + "Consume",
+					healthCheckTimeOut,
+					future -> {
+						healthCheckHandler.consumerHealthHandler(future, vertx, super.mosipEventBus, consumeAddress);
+					});
+		}
+		if (servletPath.contains("manual")) {
+			healthCheckHandler.register(
+					servletPath.substring(servletPath.lastIndexOf("/") + 1, servletPath.length()) + "Verticle",
+					healthCheckTimeOut,
+					future -> healthCheckHandler.senderHealthHandler(future, vertx, super.mosipEventBus, sendAddress));
+		}
+		if (servletPath.contains("abismiddleware")) {
+			healthCheckHandler.register("queuecheck", healthCheckTimeOut,
+					future -> healthCheckHandler.queueHealthChecker(future, mosipQueueManager, mosipConnectionFactory));
+			healthCheckHandler.register(
+					servletPath.substring(servletPath.lastIndexOf("/") + 1, servletPath.length()) + "Verticle",
+					healthCheckTimeOut,
+					future -> healthCheckHandler.consumerHealthHandler(future, vertx, super.mosipEventBus,
+							consumeAddress));
+		}
+		if (servletPath.contains("sender")) {
+			healthCheckHandler.register(
+					servletPath.substring(servletPath.lastIndexOf("/") + 1, servletPath.length()) + "Verticle",
+					healthCheckTimeOut,
+					future -> healthCheckHandler.consumerHealthHandler(future, vertx, super.mosipEventBus,
+							consumeAddress));
+		}
+
+		healthCheckHandler.register("diskSpace", healthCheckTimeOut, healthCheckHandler::dispSpaceHealthChecker);
+		healthCheckHandler.register("db", healthCheckTimeOut, healthCheckHandler::databaseHealthChecker);
+	}
+
+	private boolean checkServletPathContainsCoreProcessor(String servletPath) {

```

Comment:
I believe we renamed this method, how is the change missed ?

Reply:
Fixed